docs/setup_cluster.md

```diff
@@ -50,6 +50,7 @@ mDNS name resolution can be verified on systemd-based Linux systems using the co
 | dns_search_domains  | str  | The search domain that the DNS Service will use to resolve hostnames that are not fully qualified.  |         |    None     |
 | dns_nameservers  | list  | IPv4 addresses of DNS servers.  |         |    [8.8.8.8]     |
 | ntp_servers  | list  | FQDN or IPv4 address of a network time protocol (NTP) server.  |         |    [pool.ntp.org]     |
+| pre_cdm5  | bool  | Flag to determine if bootstrap will be on a pre-cdm 5.0 version. |         |    False     |
```

Comment:
Similar to above ... we should be able to remove the `pre_cdm5` documentation here.